usaspending_api/api_contracts/contracts/v2/disaster/agency/count.md

```diff
@@ -0,0 +1,100 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Count of Agencies receiving disaster/emergency funding [/api/v2/disaster/agency/count/]
+
+This endpoint provides the count of Agencies which received disaster/emergency funding per the requested filters.
+
+## POST
+
+This endpoint returns a count of Agencies
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes
+        + `filter` (required, Filter, fixed-type)
+
+    + Body
+
+            {
+                "filter": {
+                    "fiscal_year": 2020,
+                    "def_codes": ["L", "M", "N"]
+                }
+            }
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `count` (required, number)
+    + Body
+
+            {
+                "count": 5
+            }
+
+# Data Structures
+
+## Filter (object)
++ `def_codes` (required, array[DEFC], fixed-type)
++ `fiscal_year` (required, number)
```

Comment:
Previous versions of the mocks had a fiscal year filter, which is why this filter originated. There is no value including it for July with less than a single fiscal year, so it was pulled from the mocks. However, past discussions w/ Ross led Lizzie and I to expect that a fiscal year filter would be released between July and Nov 2020 so it was included since it would save time to build it now instead of adding it later.

All that ^^^ is old (as of 6/5/20) and I can see the benefits of having no time period filter and reporting all time for all visualizations. I would like confirmation from PO and U/X of this direction before taking the time to extract the filter.

Reply:
Removed all fiscal year filters from the API contracts